From 3ce41a9c0e3c0db0f74ca713b0595718f23ae219 Mon Sep 17 00:00:00 2001
From: Thomas Kemmer <thomas@binaryden.de>
Date: Wed, 8 Jan 2025 14:28:35 +0100
Subject: [PATCH] Use dev-lang/sassc instead of dev-python/libsass

dev-python/libsass does not currently provide keywords for riscv and
loong
---
 data/theme/meson.build   | 2 +-
 data/theme/parse-sass.sh | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/theme/meson.build b/data/theme/meson.build
index b02e5c18b..ff2909389 100644
--- a/data/theme/meson.build
+++ b/data/theme/meson.build
@@ -1,3 +1,3 @@
-sassc = find_program('pysassc')
+sassc = find_program('sassc')
 
 run_command('parse-sass.sh', check: true)
diff --git a/data/theme/parse-sass.sh b/data/theme/parse-sass.sh
index 331397115..aa75b730e 100755
--- a/data/theme/parse-sass.sh
+++ b/data/theme/parse-sass.sh
@@ -1,3 +1,3 @@
 #! /bin/bash
 
-pysassc ./cinnamon-sass/cinnamon.scss cinnamon.css
+sassc ./cinnamon-sass/cinnamon.scss cinnamon.css
-- 
2.45.2

